### PR TITLE
fix: fix arguments for org.gradle.jvmargs=-Dkotlin.daemon.jvm.options

### PR DESCRIPTION
**Pull request by PetrakovichVictoria**

### CODE REVIEW

**Comment by PetrakovichVictoria** [comment]
@Tapchicoma wait, I'll add more about the rules of - and ,.

**Comment by PetrakovichVictoria** [comment]
@Tapchicoma it's ready for review now.

File: docs/topics/gradle.md [hunk 1]
```diff
@@ -877,9 +877,16 @@ Each of the options in the following list overrides the ones that came before it
 * If the Gradle daemon's JVM arguments have the `kotlin.daemon.jvm.options` system property – use it in the `gradle.properties` file:
 
  ```properties
-  org.gradle.jvmargs=-Dkotlin.daemon.jvm.options=-Xmx1500m -Xms=500m
+  org.gradle.jvmargs=-Dkotlin.daemon.jvm.options=-Xmx1500m,Xms=500m
   ```
 
+  > In this case, enumerate arguments according to the following rules:
```
Comment (p7nov) [inline docs/topics/gradle.md:883]: Three rules in a row; maybe use a bullet list?

File: docs/topics/gradle.md [hunk 2]
```diff
@@ -877,9 +877,16 @@ Each of the options in the following list overrides the ones that came before it
...
+  > Use the minus sign `-` before the argument `Xmx` and don't use it before all other arguments.
```
Comment (p7nov) [inline docs/topics/gradle.md:884]: Does it depend on the args order? What if Xmx doesn't come first?

File: docs/topics/gradle.md [hunk 3]
```diff
@@ -877,9 +877,16 @@ Each of the options in the following list overrides the ones that came before it
...
+  > Separate all arguments with the comma ','.
```
Comment (p7nov) [inline docs/topics/gradle.md:885]:
```suggestion
  > Separate arguments with commas (`,`).
```
shorter is better

File: docs/topics/gradle.md [hunk 4]
```diff
@@ -877,9 +877,16 @@ Each of the options in the following list overrides the ones that came before it
...
+  > Don't use spaces between arguments, it will lead to the usage of the arguments after a space for the Gradle daemon, not for the Kotlin daemon.
```
Comment (p7nov) [inline docs/topics/gradle.md:886]: Isn't this a part of the second rule?
"Separate arguments with commas (`,`) _without spaces_. Arguments that come after a space will be used for .."

**Review by p7nov — COMMENTED** [review]
Some questions from my side

File: docs/topics/gradle.md (hunk 1 above)
Comment (p7nov) [reply docs/topics/gradle.md:883]: Also, I'd suggest writing this simpler and shorter. Smth like:
"When passing the arguments, follow these rules:"

File: docs/topics/gradle.md (hunk 2 above)
Comment (PetrakovichVictoria) [reply docs/topics/gradle.md:884]: No, it doesn't.

**Review by p7nov — APPROVED** [review]
Nice!

**Review by nikpachoo — APPROVED** [review]

**Pull request merged**